public/frontend/js/main.js

```
@@ -1,194 +1,196 @@
+            if (remain == 1 && oldValue == 1) {
+                $(".addMoreBtn").hide();
                 return;
             }
             var newVal = parseFloat(oldValue) + 1;
             if (remain == newVal) {
-                $('.addMoreBtn').hide();
+                $(".addMoreBtn").hide();
             }
         } else {
             if (oldValue > 0) {
                 var newVal = parseFloat(oldValue) - 1;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'newVal'. Consider to delete it or rename it.

'newVal' is defined here:


(lint/suspicious/noRedeclare)